src/components/executive-education-2u/components/EnrollmentCompletedSummaryCard.jsx

```diff
@@ -26,37 +27,80 @@ const EnrollmentCompletedSummaryCard = ({
       </Col>
       <Col xs={12} md={9}>
         <Card.Section>
-          <div className="h3 mb-4">What happens next?</div>
+          <div className="h3 mb-4">
+            <FormattedMessage
+              id="executive.education.external.course.enrollment.completed.page.summary.card.title"
+              defaultMessage="What happens next?"
+              description="Title for the executive education course enrollment completed page summary card"
+            />
+          </div>
           <div className="mb-3.5">
-            <div className="mb-1.5 text-black-color">Notified by email</div>
+            <div className="mb-1.5 text-black-color">
+              <FormattedMessage
+                id="executive.education.external.course.enrollment.completed.page.summary.card.course.notified.about.email.section.heading"
+                defaultMessage="Notified by email"
+                description="heading for the section about being notified by email to the learner"
+              />
+            </div>
             <div className="small mb-2 text-gray-500">
-              GetSmarter will email you when your course starts. Alternatively, you can visit your{' '}
-              <Hyperlink
-                destination={(features.FEATURE_ENABLE_TOP_DOWN_ASSIGNMENT && isCourseAssigned)
-                  ? dashboardUrl : externalDashboardUrl}
-                target="_blank"
-              >
-                {(features.FEATURE_ENABLE_TOP_DOWN_ASSIGNMENT && isCourseAssigned) ? 'edX dashboard' : 'GetSmarter learner dashboard'}
-              </Hyperlink>
-              {' '}for course status updates.
+              <FormattedMessage
+                id="executive.education.external.course.enrollment.completed.page.summary.card.email.section.content"
+                defaultMessage="GetSmarter will email you when your course starts. Alternatively, you can visit your <a>{dashboardHyperText}</a> for course status updates."
+                description="Content for the section about being notified by email to the learner with a respective dashboard link"
+                values={{
+                  // eslint-disable-next-line react/no-unstable-nested-components
+                  a: (chunks) => (
+                    <Hyperlink
+                      destination={(features.FEATURE_ENABLE_TOP_DOWN_ASSIGNMENT && isCourseAssigned)
+                        ? dashboardUrl : externalDashboardUrl}
+                      target="_blank"
+                    >
+                      {chunks}
+                    </Hyperlink>
+                  ),
+                  dashboardHyperText: (features.FEATURE_ENABLE_TOP_DOWN_ASSIGNMENT && isCourseAssigned) ? (
+                    'edX dashboard'
+                  ) : (
+                    'GetSmarter learner dashboard'
+                  ),
+                }}
+              />
```

Comment:
1. I think we need to use multiple FormattedMessage or intl.formatMesasge
2. We need to mention that `edX` and `GetSmarter` are brand names so that translator handle them accordingly